package-release.sh

```diff
@@ -56,16 +56,19 @@ function prepare {
     src/nvapi_d3d12.cpp       \
     src/nvapi_interface.cpp   \
     external/nvapi/nvapi_interface.h
-
-  # remove existing version.h, because otherwise the existing one gets into the build instead of the generated one
-  if [ -e version.h ]; then
-    rm version.h
-  fi
 }
 
 function build_arch {
   cd "$SRC_DIR"
 
+  # remove generated files, because otherwise the existing one gets into the build instead of the generated one
+  if [ -e version.h ]; then
+    rm version.h
+  fi
+  if [ -e config.h ]; then
+    rm config.h
+  fi
```

Comment:
I'd simplify this to just
```suggestion
  rm -f config.h version.h
```
which I believe is a bit more idiomatic for shell scripts.